processing/data_collection/gazette/spiders/rj_sao_goncalo.py

```diff
@@ -0,0 +1,28 @@
+from gazette.items import Gazette
+import datetime
+from gazette.spiders.base import BaseGazetteSpider
+
```

Comment:
We can follow [PEP8 guideline for imports](https://www.python.org/dev/peps/pep-0008/#imports) here, so:
```
import datetime

from gazette.spiders.base import BaseGazetteSpider
from gazette.items import Gazette
```